drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyConnectionFactoryConfig.java

```diff
@@ -63,6 +64,19 @@ public DriftNettyConnectionFactoryConfig setConnectionPoolEnabled(boolean connec
         return this;
     }
 
+    @Min(1)
+    public int getConnectionPoolMaxConnections()
+    {
+        return connectionPoolMaxConnections;
+    }
+
+    @Config("thrift.client.connection-pool.max-connections")
```

Comment:
Now it's confusing compared to `max-size` :-) Both look like they are limits on the total number of connections in the pool.

This is a per endpoint property so it really does have to include that IMO.